modules/storage-rocksdb/src/test/java/org/apache/ignite/internal/storage/rocksdb/RocksDbTableStorageTest.java

```diff
@@ -73,21 +75,27 @@ public void setUp(
             @InjectConfiguration RocksDbStorageEngineConfiguration rocksDbEngineConfig,
             @InjectConfiguration(
                     name = "table",
-                    polymorphicExtensions = {HashIndexConfigurationSchema.class, RocksDbDataStorageConfigurationSchema.class}
+                    polymorphicExtensions = {
+                            HashIndexConfigurationSchema.class,
+                            UnknownDataStorageConfigurationSchema.class,
+                            RocksDbDataStorageConfigurationSchema.class
+                    }
             ) TableConfiguration tableCfg
     ) throws Exception {
-        assertThat(tableCfg.dataStorage(), is(instanceOf(RocksDbDataStorageConfiguration.class)));
+        CompletableFuture<Void> changeDataStorageFuture = tableCfg.dataStorage().change(c -> c.convert(RocksDbDataStorageChange.class));
```

Comment:
In previous test you just wait 1 second, but here you assert that it'll give you null eventually. Why?

Reply:
Also done below, can make a wait of 1 second. WDYT?

Reply:
I don't mind any solution, just wondering why it's inconsistent

Reply:
I didn’t quite understand, but if it’s a minor, then we’ll leave it